.github/pull_request_template.md

```diff
@@ -13,7 +13,7 @@ The following selections do not need to be completed if this PR only contains ch
   - [ ] TODO (before merge)
   - [ ] Tests included
   - [ ] Tests not necessary because:
-- Wrangler E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
+- Wrangler / Vite E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
```

Comment:
why make this distinction?

Reply:
I read `Wrangler / Vite` as wrangler or vite tests, which is why I was a bit confused. But that might be me missinterpreting that `/`? in which case ignore my comment